README.md

```diff
@@ -3,43 +3,47 @@
 Bridge selected events from AppStudio into [Segment][1]
 
 ```mermaid
-flowchart TD
+flowchart TB
     subgraph A["RHTAP clusters"]
         A1[(API server)]
-        A2(["OpenShift
-         logging collector"])
+        A2(["`OpenShift
+        logging collector`"])
```

Comment:
What's the meaning/purpose of adding the ` accent?

Reply:
It makes the text be processed as markdown, but unfortunately this is not supported by the Mermaid version used by GitHub, so removed.